subsys/bluetooth/audio/cap_stream.c

```diff
@@ -25,9 +26,9 @@
 
 LOG_MODULE_REGISTER(bt_cap_stream, CONFIG_BT_CAP_STREAM_LOG_LEVEL);
 
-static bool is_cap_initiator_unicast(struct bt_bap_stream *bap_stream)
+static bool stream_is_central(struct bt_bap_stream *bap_stream)
 {
-	if (IS_ENABLED(CONFIG_BT_CAP_INITIATOR) && IS_ENABLED(CONFIG_BT_BAP_UNICAST_CLIENT)) {
+	if (IS_ENABLED(CONFIG_BT_CONN)) {
```

Comment:
just for my understanding:

Is it any specific reason we did not modify the function like:

if (IS_ENABLED(CONFIG_BT_CAP_INITIATOR) && IS_ENABLED(CONFIG_BT_BAP_UNICAST_CLIENT) && IS_ENABLED(CONFIG_BT_CONN))
{
......
}

then all the following place, we can just call one function instead of "(IS_ENABLED(CONFIG_BT_CAP_INITIATOR) && IS_ENABLED(CONFIG_BT_BAP_UNICAST_CLIENT) && stream_is_central(bap_stream))"

Reply:
Yes: It's because that does not work for builds for x86: https://github.com/zephyrproject-rtos/zephyr/actions/runs/9357616500/job/25825761204?pr=72075

Not sure exactly why, but this is blocking CAP development so rather get a quick fix in rather than trying to fix the build system/compiler :)

Reply:
Sounds like there is some GCC 12 vs 13 things going on right now, so that is probably why